tests/Oryx.Integration.Tests/NodeEndToEndTests.cs

```diff
@@ -34,6 +34,7 @@ public NodeEndToEndTestsBase(ITestOutputHelper output, TestTempDirTestFixture te
             DockerVolume.CreateMirror(Path.Combine(_hostSamplesDir, "nodejs", appName));
     }
 
+    [Trait("category", "node")]
```

Comment:
Oh, so these tests were not running for few days/weeks now...could you please check the same for other platform tests too?